x-pack/plugins/security_solution/public/resolver/store/middleware.ts

```diff
@@ -92,6 +93,31 @@ export const resolverMiddlewareFactory: MiddlewareFactory = (context) => {
           });
         }
       }
+    } else if (
+      (action.type === 'userRequestedRelatedEventData' ||
+        action.type === 'appDetectedMissingEventData') &&
+      context
+    ) {
+      const entityIdToFetchFor = action.payload;
+      let result: ResolverRelatedEvents;
+      try {
+        result = await context.services.http.get(
+          `/api/endpoint/resolver/${entityIdToFetchFor}/events`,
+          {
+            query: { events: 100 },
+          }
+        );
+
+        api.dispatch({
+          type: 'serverReturnedRelatedEventData',
+          payload: result,
+        });
+      } catch (e) {
+        api.dispatch({
+          type: 'serverFailedToReturnRelatedEventData',
```

Comment:
I don't see this being handled. We could add this as an enhancement. TODO, file a ticket for this (if we care.)